src/components/Header/Header.js

```diff
@@ -1,20 +1,25 @@
-import React from 'react'
-import './Header.css';
-import HeaderMenu from '../HeaderMenu/HeaderMenu';
+import React from "react";
+import "./Header.css";
+import HeaderMenu from "../HeaderMenu/HeaderMenu";
+
 
 export default function Header() {
-    return (
-        <div className="lqe-hdr">
-            <div className='lqe-top-navbar-header'>
-                <div className="lqe-left-top-navbar-sec">
-                LIFE DEMO
-                </div>
-                <div className="lqe-right-top-navbar-sec">
-                   <div className="lqe-left-sec"><a className="lqe-phone-link" href="tel:+1-51111-111">051111-111</a></div>
-                   <div className="lqe-right-sec"><button className="lqe-my-life-button" type="button"></button></div>
-                </div>
-            </div>
-            <HeaderMenu></HeaderMenu>
+  return (
+    <div className="lqe-hdr">
+      <div className="lqe-top-navbar-header">
```

Comment:
More of a question than an issue. Is this following BEM? Should it be lqe-top-navbar-header really be lqe-hdr-top-navbar and similar for the class names that follow this one?

Reply:
This wasn't my file - but I removed some stuff that Xin added and then formatted it to 2 spaces - I don't think it is following BEM.  Should it not be _lqe-top-navbar__header_ ?